src/net/sourceforge/kolmafia/MonsterData.java

```diff
@@ -87,10 +87,6 @@ public String getOption() {
     }
   }
 
-  static {
-    Attribute[] attributes = Attribute.values();
```

Comment:
I believe this was here to call the constructor for every attribute, to populate `optionToAttribute`.

Is it unnecessary?

Reply:
Yeah, the constructors are called when the enum is loaded by the ClassLoader

Reply:
That is not correct.

- I commented out that line and recompiled.
- logged in a character.
- gCLI reports this:
```
Monster: "1335 HaXx0r": unknown option: Atk:
Monster: "1335 HaXx0r": unknown option: 81
Monster: "1335 HaXx0r": unknown option: Def:
Monster: "1335 HaXx0r": unknown option: 72
Monster: "1335 HaXx0r": unknown option: HP:
Monster: "1335 HaXx0r": unknown option: 71
Monster: "1335 HaXx0r": unknown option: Init:
Monster: "1335 HaXx0r": unknown option: 50
Monster: "1335 HaXx0r": unknown option: Meat:
Monster: "1335 HaXx0r": unknown option: 30
Monster: "1335 HaXx0r": unknown option: P:
Monster: "1335 HaXx0r": unknown option: dude
Monster: "1335 HaXx0r": unknown option: EA:
Monster: "1335 HaXx0r": unknown option: "bad
Monster: "1335 HaXx0r": unknown option: spelling"
Monster: "1335 HaXx0r": unknown option: Article:
Monster: "1335 HaXx0r": unknown option: a
```

Reply:
@midgleyc @Veracity0 hmm. ok, fixed it I think